src/App/Mail/SendTodoDeadlineExpiredMailSubscriber.php

```diff
@@ -0,0 +1,54 @@
+<?php
+namespace Prooph\ProophessorDo\App\Mail;
+
+use Prooph\ProophessorDo\Model\Todo\Event\TodoWasMarkedAsExpired;
+use Prooph\ProophessorDo\Projection\Todo\TodoFinder;
+use Prooph\ProophessorDo\Projection\User\UserFinder;
+use Zend\Expressive\Template\TemplateRendererInterface;
+use Zend\Mail\Transport\TransportInterface;
+use Zend\Mail;
+
+/**
+ * Class SendTodoDeadlineExpiredMailSubscriber
+ *
+ * @package Prooph\ProophessorDo\App\Mail
+ * @author Michał Żukowski <michal@durooil.com
+ */
+final class SendTodoDeadlineExpiredMailSubscriber
```

Comment:
I think that class should be moved, too and also send a command through the command bus, instead of sending the mail right away.
Like SendEmailThatXYZWasChangeDToUserABC